docs/_snippets/features/mathtype.js

```diff
@@ -62,6 +62,17 @@ ClassicEditor
 		},
 		cloudServices: CS_CONFIG
 	} )
+
+	// added code
```

Comment:
The convention for comments is to start them as a sentence from an uppercase letter and end with a period.

However, this is not a legit code comment anyway. I think it's some kind of a leftover.

Reply:
I cannot see any of these comments in the files anymore. These have been removed before final commit.